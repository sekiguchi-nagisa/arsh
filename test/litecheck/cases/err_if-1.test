#!/bin/sh

echo 12345 1>&2 && echo ACBD 1>&2
# CHECKERR: @@@@@