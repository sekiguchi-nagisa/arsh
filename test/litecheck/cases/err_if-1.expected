CHECKERR on line 4,
1c1,2
< @@@@@
---
> 12345
> ACBD
stderr is mismatched